sources/packages/web/src/components/generic/BodyHeader.vue

```diff
@@ -1,18 +1,20 @@
 <template>
-  <div class="mb-4">
-    <span class="category-header-large color-blue">
+  <v-row no-gutters>
+    <v-col class="category-header-large color-blue" order="1">
```

Comment:
What is the benefit of using the `order` here if we are not changing the order?

Reply:
https://next.vuetifyjs.com/en/components/grids/#order. This was part of my experiment. ya in this case there is no need for the order, I will remove it.
![image](https://user-images.githubusercontent.com/77353155/180503466-f47ba9a1-cf67-49c0-bffc-e79710cb6552.png)